apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -16,9 +16,11 @@
 #import <React/UIView+React.h>
 
 #ifdef RCT_NEW_ARCH_ENABLED
+#import <React/RCTParagraphComponentView.h>
```

Comment:
Are changes on iOS still required with this approach?

Reply:
I'm not sure what do you mean 😅

Reply:
Since we are relying on the Text's `onPress` to trigger event, do we need to update the native recognizer to check for text?

Reply:
Seems like we don't have to. Removed in [65ea0f9](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/65ea0f9bc1da6f55b45b7ba60f570d3f545f58f7)